### PR TITLE
Cheesy Topic

### PR DESCRIPTION
**Pull request by Danymations**
I thought it would be a bit cute to add a small conversation, haha.

### CODE REVIEW

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -4344,6 +4344,28 @@ label monika_confidence_2:
     m "Your everlasting love and care is just about all the support I need in order get to where I want to be."
     m "What kind of girlfriend would I be if I didn't return the favor~?"
     return
+    
+    init 5 python:
```
Comment (ThePotatoGuy) [inline Monika After Story/game/script-topics.rpy]: Remove the indentation in front of the `init` block. `init` python blocks must **not** have any indentation.

**Comment by Danymations** [comment]
Done, thanks!

**Comment by Rune0n** [comment]
Honestly, there's potential here, but the topic seems underdeveloped. A re-write to include a few more "cheesy" cliches, as well as extra attention paid to organization, would be appreciated.

**Comment by Danymations** [comment]
Will do!

**Pull request closed without merging**